bin/MadGraph5_aMCatNLO/gridpack_generation.sh

```diff
@@ -595,6 +603,7 @@ make_gridpack () {
         $WORKDIR/$MGBASEDIRORIG/MadSpin/madspin madspinrun.dat 
         rm madspinrun.dat
         rm -rf tmp*
+        cp $CARDSDIR/${name}_madspin_card.dat $WORKDIR/process/madspin_card.dat
```

Comment:
Is this to keep the card once more in madspin directory although it's not 100% needed? or?

Reply:
Can probably remove